src/extension.ts

```diff
@@ -1620,7 +1620,7 @@ export async function activate(
 
     // deferred activation in case activationCfg is set by peer extension synchronously
     setTimeout(async () => {
-        await zephyr.activate(api.activationCfg.topdir);
+        await zephyr.activate(api.activationCfg?.topdir);
```

Comment:
Should rather be handled in the type definition for the API class, or make sure it's always defined.